cardano-prelude-test/cardano-prelude-test.cabal

```diff
@@ -147,4 +109,4 @@ test-suite cardano-prelude-test
                        -fno-warn-unused-packages
```

Comment:
Any reason to disable this warning? (Not specific to this PR) I would leave it enabled wherever I can.

Reply:
Our policy is to disable `8.10` warnings until after we switch, after which we will clean this up.

Reply:
Sure, I understand that for the other warnings, because fixing them for 8.10 might trigger other warnings for 8.6. But that's not the case for this warning.

Reply:
This particular one turned out to be un-intrusive so I will re-enable it.